support-frontend/assets/helpers/tracking/acquisitions.js

```diff
@@ -261,11 +262,22 @@ function deriveSubsAcquisitionData(
 
 }
 
+function deserialiseAusMomentAcquisitionData(serialised: string): Object {
```

Comment:
return type can be `ReferrerAcquisitionData`

Reply:
@tomrf1 the result of calling this function or `deserialiseJsonObject` is stored as `paramData`. In turn, this is passed into `buildReferrerAcquisitionData`, which expects a generic object as its argument. As the new `deserialiseAusMomentAcquisitionData` function should be able to replace the default `deserialiseJsonObject` function, I wanted to make their signatures as similar as possible. 

Are you happy with all this or would you like me to change the return type still? It's totally doable, it just means a few changes in different places.

Reply:
👍